Makefile

```diff
@@ -118,6 +117,8 @@ lint:
 
 clean:
 	$(SUDO) docker rmi $(IMAGE_NAMES) >/dev/null 2>&1 || true
+	$(SUDO) docker rm -f $(docker ps -a | grep centos-multimaster-node) >/dev/null 2>&1 || true
+	$(SUDO) docker rm -f $(docker ps -a | grep ubuntu-multimaster-node) >/dev/null 2>&1 || true
```

Comment:
A nit, but it would probably be nicer to put these together:

`$(SUDO) docker rm -f $(docker ps -a | egrep '(centos|ubuntu)-multimaster-node') >/dev/null 2>&1 || true`

Reply:
👍

Reply:
Fixed.